types/angular-cookies/index.d.ts

```diff
@@ -67,7 +86,7 @@ declare module 'angular' {
 
         /**
          * CookieStoreService DEPRECATED
-         * see https://code.angularjs.org/1.2.26/docs/api/ngCookies/service/$cookieStore
+         * see https://code.angularjs.org/1.3.20/docs/api/ngCookies/service/$cookieStore
```

Comment:
AFAIK the notion of store needs to be completely removed in order to match 1.8:
https://github.com/angular/angular.js/blob/e55d352e942465479fa9f93b566db20a96b4cf15/docs/content/guide/migration.ngdoc#L453
There is only single public api now for cookies:
https://github.com/angular/angular.js/blob/master/src/ngCookies/cookies.js
https://code.angularjs.org/1.8.0/docs/api/ngCookies/service/$cookies